### DISCUSSION TITLE
🚀 Feature Request: Support `fail_open` options on `routes`.

### DISCUSSION THREAD

**Discussion by GregBrimble** [cloudflare/workers-sdk#13299] [Feature Requests]
### Describe the solution

When I have the free plan, on the dashboard I can set my Worker route to fail open or fail closed (falling back to the host origin) when I hit my free invocation limits. We should consider offering this as a configuration option in Wrangler too.

**Comment by SteveLeve** [comment]
This is a very helpful option for rearchitecture of legacy apps. +1
